src/hermes/commands/marketplace.py

```diff
@@ -16,7 +16,7 @@
 from hermes.commands.init.util import slim_click
 from hermes.utils import hermes_doi, hermes_concept_doi, hermes_user_agent
 
-MARKETPLACE_URL = "https://hermes.software-metadata.pub/marketplace"
+MARKETPLACE_URL = "https://docs.software-metadata.pub/en/latest/plugins/marketplace.html"
```

Comment:
This URL must not be changed! Instead, the redirect `/marketplace` on Read the Docs must be changed to point to the new place where the plugins can be found. This indirection exists so that we don't break `hermes-marketplace` commands of earlier HERMES versions.

Reply:
(Maybe this could be added as a comment in the source code, while you're at it 😄)